tests/features/user/register.feature

```diff
@@ -3,12 +3,6 @@ Feature: User registration
   As a user I must be able to register to the site and complete my user profile
   and receive appropriate notifications.
 
-  Scenario: User can find the register page
-    Given I am an anonymous user
-    When I am on the homepage
-    And I click "Sign in (legacy)"
-    Then I should see the heading "Sign in"
-
```

Comment:
Not true. This will still work as, we've decided in standup.

Reply:
Will add it back and add a `@wip` tag and a `Todo` with a mention of ISAICP-5761. I don't even remember this being discussed in the standup, but it makes a lot of sense. It would be better to write down these decisions in the ticket instead of relying verbal agreements and people's bad memory ;)